migrations/db_schema_version.go

```diff
@@ -12,3 +12,11 @@ var dbSchemaVersion = Migration{
 		return CommitProgress(db, nil, true)
 	},
 }
+
+var dbSchemaVersion2 = Migration{
+	Name: "db_schema_version2",
+	Up: func(db ethdb.Database, tmpdir string, progress []byte, CommitProgress etl.LoadCommitHandler) (err error) {
+		// This migration is no-op, but it forces the migration mechanism to apply it and thus write the DB schema version info
```

Comment:
I think this migration should fail for MDBX if it uses old table names

Reply:
Or perhaps let's not bother. But I would prefer that instead of this migration we have tool in integration to force version update for existing DB (only for people who know what they are doing)

Reply:
In fact, we don't need a special command, it is already possible to do by manipulating migrations